docs/Configure-GenericAptRepo.md

```diff
@@ -0,0 +1,23 @@
+# How to configure a generic aptitude repo
```

Comment:
repo -> repository

Put also a link with official docs on what is an aptitude repo